packages/docs/src/stories/components/menu-item.stories.ts

```diff
@@ -0,0 +1,67 @@
+import '../../../../components/src/solid-components';
+import { html } from 'lit-html';
+import { storybookDefaults, storybookHelpers, storybookTemplate } from '../../../scripts/storybook/helper';
+import { withActions } from '@storybook/addon-actions/decorator';
+
+const { argTypes, parameters } = storybookDefaults('sd-menu-item');
+const { overrideArgs } = storybookHelpers('sd-menu-item');
+const { generateTemplate } = storybookTemplate('sd-menu-item');
+
+export default {
+  tags: ['!dev', 'skip-a11y-[aria-required-parent]'],
```

Comment:
In the storybook 9, it is required to have `autodocs` to generate the docs, so lets have it already to support storybook 9

```suggestion
  tags: ['!dev', 'autodocs', 'skip-a11y-[aria-required-parent]'],
```